### ISSUE TITLE
Expose nexus logs in the Taskcluster task called `toolchain-linux64-android-gradle-dependencies`

### ISSUE THREAD

**Issue by JohanLorenzo** [mozilla-mobile/fenix#16648]
Today, the toolchain task in charge downloading **all** maven/gradle dependencies fails[1]. The only error we get is:

```
[task 2020-11-19T12:48:35.174Z] > Could not find jna-5.6.0.aar (net.java.dev.jna:jna:5.6.0).
[task 2020-11-19T12:48:35.174Z] Searched in the following locations:
[task 2020-11-19T12:48:35.174Z] http://localhost:8081/nexus/content/repositories/jcenter/net/java/dev/jna/jna/5.6.0/jna-5.6.0.aar
```

It's sadly not enough to be actionable. Under the hood, this task uses Nexus, the well-known maven cache. In our setup, Nexus outputs logs at `/opt/sonatype/nexus/logs`. We can simply get more information by adding 

```
        artifacts:
            - type: directory
              name: public/logs/nexus
              path: /opt/sonatype/nexus/logs
```

under this section https://github.com/mozilla-mobile/fenix/blob/9cdfb6db4a2eb67e83359ac5ac545fa4f7d81509/taskcluster/ci/toolchain/android.yml#L52

This was already tested back in May[2]. We got this kind of helpful log lines[3]:  

```
jvm 1    | 2020-05-14 16:15:56,723+0000 INFO  [qtp152814843-88] anonymous org.sonatype.nexus.proxy.maven.maven2.M2Repository - Proxied item jcenter:/org/jetbrains/kotlinx/kotlinx-metadata-jvm/0.0.5/kotlinx-metadata-jvm-0.0.5.jar evaluated as INVALID during content validation (validator=checksum, sourceUrl=https://jcenter.bintray.com/org/jetbrains/kotlinx/kotlinx-metadata-jvm/0.0.5/kotlinx-metadata-jvm-0.0.5.jar)
```

For the record, android-components, reference-browser, and geckoview can all benefit from this change too.

This is a simple change to make. I am happy to mentor it.

[1] https://treeherder.mozilla.org/jobs?repo=fenix&group_state=expanded&resultStatus=testfailed%2Cbusted%2Cexception%2Cretry%2Cusercancel%2Crunnable&revision=9cdfb6db4a2eb67e83359ac5ac545fa4f7d81509&selectedTaskRun=dsjUbynmTDO0Xme3RHITGQ.0
[2] https://github.com/mozilla-mobile/android-components/compare/master...pocmo:build-testing#diff-03111c8f844b65efbfd39751c9dce4253049081ba9dfa0a0e5f3b21b3c402f2bR56
[3] https://firefox-ci-tc.services.mozilla.com/tasks/I4LtilWURIaCgBD1DKME-w/runs/1/logs/https%3A%2F%2Ffirefox-ci-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FI4LtilWURIaCgBD1DKME-w%2Fruns%2F1%2Fartifacts%2Fpublic%2Flogs%2Fnexus%2Fwrapper.log#L209

**Comment by mnbpdx** [comment]
Hey @JohanLorenzo, I'm pretty green but really excited to get started contributing to open source. Would you willing to mentor me through this?

**Comment by mnbpdx** [comment]
I went ahead and forked, branched, and added the code, is this correct?

<img width="575" alt="image" src="https://user-images.githubusercontent.com/19321892/99915900-7e09a580-2cbb-11eb-9df5-bf73acbf4b25.png">

Let me see if I can explain what this does:

Nexus is a repository manager, that Firefox uses as like a layer between it and Maven. It failed when downloading some maven/gradle stuff. The error we got back was unhelpful. The code I just added outputs Nexus logs, which will better explain what went wrong.

Is that sort of correct?
